src/test/java/com/google/javascript/gents/singleTests/comments.ts

```diff
@@ -29,14 +29,16 @@ let foo = function(deleted: number, notdeleted: number): number {
 };
 
 // The following comments should be mostly deleted
-/**
```

Comment:
Multi-line `@type` annotation matched 
`
/**
  * @type {number}
  * @private {number}
  * @protected {number}
  * @param {number} foo description of foo
  * @public {number}
  * @package {number}
  */
`
as a whole. I'm excluding `@` in the body. I think it should resolve the problem?